imports/client/components/SetupPage.tsx

```diff
@@ -48,14 +54,17 @@ const SectionHeaderButtons = styled.span`
   flex: 0 0 auto;
   button {
     margin-left: 8px;
-    margin-bottom: 8px;
   }
 `;
 
+const SectionContent = styled.div`
+  max-width: 800px;
+`;
+
```

Comment:
I'm definitely not wild about the section content being max-width'd but the headers stretching all the way to the end of the screen - especially with the feature flag buttons, it looks kind of imbalanced on my screen:

<img width="1912" alt="Screen Shot 2022-03-23 at 8 34 44 PM" src="https://user-images.githubusercontent.com/28167/159836969-dc05309b-e647-4604-90df-e1fe551bfb79.png">

I think I'd rather just max-width the whole page?

Reply:
Fair.  My first experiment with that approach felt like it was hard to pick out the sections at a glance, but 1) that was before I had worked out negative margins to extend the section headers into the gutter, and 2) I agree this looks sorta ridiculous at very large widths.

Also, in the fullness of time, it might be nice to have a miniature navbar/scrollspy thing at large screen sizes since we'll have the space for it and the setup page is super long.